### PR TITLE
Create first page

### PR DESCRIPTION
**Pull request by DraganU**

### CODE REVIEW

File: .browserslistrc [hunk 1]
```diff
@@ -0,0 +1,3 @@
+defaults,
```
Comment (milana986) [inline .browserslistrc:1]: Why we using this file?

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+# Optional eslint cache
+.eslintcache
+
+# Yarn Integrity file
+.yarn-integrity
+
+# Optional sass cache
+/.sass-cache
+gulpfile.tmp.*
```
Comment (milana986) [inline .gitignore:42]: Is this really necessary?

File: LICENSE.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
+MIT License
```
Comment (milana986) [inline LICENSE.md:2]: Please remove this

File: assets/img/avatars/5.png
Comment (milana986) [inline assets/img/avatars/5.png:1]: handsome guy :)

File: assets/js/main.js [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+
+'use strict';
+
+let menu, animate;
+
+(function () {
+  // Initialize menu
+  //-----------------
+
+  let layoutMenuEl = document.querySelectorAll('#layout-menu');
```
Comment (milana986) [inline assets/js/main.js:13]: Great solution. Good job!

**Review by milana986 — APPROVED** [review]

File: .eslintrc.json [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+    "arrow-parens": ["error", "as-needed"],
+    "no-param-reassign": "off",
+    "no-plusplus": [
+      "error",
+      {
+        "allowForLoopAfterthoughts": true
+      }
+    ],
+    "no-cond-assign": ["error", "except-parens"]
+  }
```
Comment (milana986) [inline .eslintrc.json:23]: jjjjj

File: .gitignore (hunk 2 above)
Comment (DraganU) [reply .gitignore:42]: This is essential part of every app...

File: LICENSE.md (hunk 3 above)
Comment (DraganU) [reply LICENSE.md:2]: I planned to remove this... Will be removed

File: assets/img/avatars/5.png
Comment (DraganU) [reply assets/img/avatars/5.png:1]: I am not sure

File: assets/js/main.js (hunk 4 above)
Comment (DraganU) [reply assets/js/main.js:13]: Tnx

**Pull request merged**